### DISCUSSION TITLE
Hi, Can i use strapi for limited role based access for documents?

### DISCUSSION THREAD

**Discussion by heyAyushh** [strapi/strapi#7564] [Legacy Topics]
ex. A author can only change his blog posts and not all the blog posts. 
ex. A restaurant owner can only change food items and restaurant details post.

**Comment by adrian-ub** [comment]
it is enough to modify the controllers, asking if the user who wants to edit said document is the owner

https://strapi.io/documentation/v3.x/guides/is-owner.html#introduction

**Reply by derrickmehaffy** [reply]
This is correct for the generated routes, the new Admin RBAC feature does have this option as well but it's not entirely configurable for the Community edition
